nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtulhost config
+FROM nginx:latest
```

Comment:
This goes with https://github.com/tgross/triton-cloudflare/pull/1/files#r43663385, but naming this directory `nginx` is confusing. Should it be `examples`?